modules/logging_configs/main.tf

```diff
@@ -0,0 +1,193 @@
+# This is just a module to provide a template for creating a log configuration for an ECS task definition.
+
+locals {
+  tld = var.region == "eu-central-1" ? "eu" : "com"
+
+  fire_lens_container = <<EOF
+  {
+    "essential": true,
+    "image": "906394416424.dkr.ecr.${var.region}.amazonaws.com/aws-for-fluent-bit:stable",
+    "name": "log_router",
+    "cpu": 0,
+    "user": "0",
+    "environment": [],
+    "volumesFrom": [],
+    "portMappings": [],
+    "mountPoints": [],
+    "firelensConfiguration": {
+      "type": "fluentbit",
+      "options": {
+        "enable-ecs-log-metadata": "true"
+      }
+    },
+    "memoryReservation": 50,
+    "dockerLabels": {
+      "com.datadoghq.tags.env": "${terraform.workspace}",
```

Comment:
@magreenbaum just want to make sure this file is good with you before I merge since it's been updated after your approval

Reply:
If we're not using that environment_name var anymore, we should remove it from variables.tf.

Reply:
yep that's the next commit

Reply:
done ✅

Reply:
👍 LGTM thanks again!!